### DISCUSSION TITLE
FILDevSummit Singapore'23: Client Data Onboarding Protocol / Tools / Market

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#663] [General]
> What's FILDev Summit ➡️ [here](https://github.com/filecoin-project/community/discussions/661).

**Track Leads:** [@xilin.eth](Xilin.eth) & Mara McMahon
Navigating enterprise deals requires thoughtful strategy and planning. In this interactive series of workshops, we will facilitate roundtable discussions to align on best practices for preparing enterprise deals and improving the user experience. Working together with ecosystem members, product managers and engineers, we will discuss deal structures, strategies for large-scale data storage and payments, and approaches to access controls and supporting deletions. This is an opportunity to cultivate shared knowledge to close and support large-scale Enterprise deals.

Full agenda ⏭️ [fildev.io](https://fildev.io/)
Subscribe this discussion to get updates from this track!

**Comment by celine3650** [comment]
Solana Client Use Case Discussion led by Yan bo:

What was shared: 
1. Walk through how to build a public archive of the Solana blockchain using the help from lead SP, replica SP and singularity for data prep. 
2. Share the future state that deepen the collaboration and have a plan for retrieval. 

Questions raised that need follow up:
1. The cost of CDN is too high specially for the asian market, what do we do about this from incentive standpoint? 
2. Replicating for Other Blockchains: The audience was interested in understanding how this process could be replicated for other blockchains and what would be required from a business development perspective.
3. There is a question about the storage requirements for snapshots and the potential value of these datasets were raised.
4. Replicators: The need for multiple SPs to become replicators was highlighted, along with the challenges of finding them. The process was manual and low efficient
5. Data Retrieval: Concerns were raised about the lack of incentives for data retrieval and the tools being developed to make retrieval profitable.

**Comment by celine3650** [comment]
**Paid deal preparedness discussion led by Mara and Porter**
1. What's blocking you to get on the paid deal path? [Top 3 things] 
- User experience 
- Fast retrieval
- Working software 
2. What's your top wish from PL?
- S3 gateway
- Cost estimator
- Native retrieval without bearing the cost of keeping unsealed copy

More details and follow ups coming soon

**Comment by celine3650** [comment]
**Data onboarding tooling discussion led by Xilin and Riba**

Shared Information:
Motion:
- Xilin shared the progress of the filecoin restful API dev work (project motion). https://github.com/orgs/filecoin-project/projects/94/
- Encouraged SPs to sign up for the motion private beta program. https://destor.io/filecoin-rest-api

Singularity:
- Xilin provided an update on singularity V2 and its website release: https://singularity.storage
- Discussed potential collaboration with lighthouse.storage regarding the singularity app. 

Spade:
- Riba outlined the components of the Spade platform, including Tenant Service, Ground Control, Retrieval Bot, Chain State Oracle, Communication/Chain Relay Service, and Broker Service.

Questions Asked:
Motion:

- What specifications are required for SPs to run motion?
- Who will handle key management for clients during the beta release?
- How will on-chain FIL payments be accessed? How will I pay SP?
- Who will manage the egress fee?
- Can I retrieve CIDs?
- How will I retrieve from SP price estimates to show to my customers

Singularity:
- Can singularity use the front-end of lighthouse.storage?
- How can lighthouse.storage benefit from singularity's scalability?

Spade:
Once ready, how open-source will the components of Spade be?

**Comment by celine3650** [comment]
**Partner spotlight from Huawei Cloud, FilSwan Lagrange Dao and Lighthouse.storage** 

1. Huawei cloud 
2. Filswan Lagrange Dao
3. Lighthouse.storage

**Reply by celine3650** [reply]
Purpose: **Lighthouse.storage** offers perpetual storage solutions, allowing users to store data long-term with a one-time fee. This can range from a few years to potentially forever.

Targeting Audience: Lighthouse primarily serves teams building on FCM through their SDK. This includes developers integrating the SDK for NFTs, data availability, and general storage.

Highlights:

**Encryption**: By default, data will soon be encrypted unless made public. This ensures enhanced security for user data.

**Integration**: Lighthouse has integrated with various products and offers features like time-based encryption and support for fingerprint and face ID for authentication.

Open Source: The underlying technology is open-source, allowing teams and developers to contribute and enhance the platform.

Questions: Can singularity benefit from the front-end of lighthouse.storage
and can LS benefit from the scalibity of singularity so they can support larger volume of data for the clients.

**Reply by celine3650** [reply]
Lagrange DAO https://lagrangedao.org/ focuses on decentralized solutions for storage, computing, and data.
1. They started as a storage provider and have since expanded their ecosystem to include data onboarding, staking services, storage DRS, and more.
2. They have IPFS gateways worldwide, ensuring efficient data retrieval and storage.
3. They offer a decentralized dataset that can be used for various applications, from AI models to gaming.
4. They have introduced a new concept of "platform provider" which combines storage, computing, and network provision.
5. They have a revenue-sharing model with storage providers, incentivizing them to join the network.
6. They have integrated with various payment methods and accept stable coins for their services.

**Reply by celine3650** [reply]
Huawei Cloud
1.  It free SP from worrying about the hardware, bandwidth, network connectivity, and maintenance requirements of running a node so they can focus on their core business instead. Greater Heat is one of their partners. They are looking to work with more SPs. 
2. They are also working with Nebula-data (an ISV) to provide a web3 S3 gateway, it's still at the early dev stage, so we might be able to  support this ISV through project motion.
